packages/exo/src/exo-tools.js

```diff
@@ -53,8 +69,8 @@ const defendSyncMethod = (method, methodGuardPayload, label) => {
   const { syncMethod } = {
     // Note purposeful use of `this` and concise method syntax
     syncMethod(...args) {
-      defendSyncArgs(harden(args), methodGuardPayload, label);
-      const result = apply(method, this, args);
+      const realArgs = defendSyncArgs(harden(args), methodGuardPayload, label);
```

Comment:
please also update the signature to qualify the args
```suggestion
      const realArgs = defendSyncArgs(harden(rawArgs), methodGuardPayload, label);
```

Reply:
I don't understand. There is no `rawArgs` in scope here. Do you suggest that I rename the syncMethod rest parameter to `rawArgs`?

> please also update the signature to qualify the args

I don't understand this either. The returned `syncMethod` as a whole is declared as `Method`. What do you have in mind?

Reply:
I believe the request is to rename `args` -> `rawArgs` to make it clearer what args represent

Reply:
right. I would have edited both lines but the "suggestion" tool only works on lines modified in the PR.
```js
    syncMethod(...rawArgs) {
      const realArgs = defendSyncArgs(harden(rawArgs), methodGuardPayload, label);
```

Reply:
Gotcha. This rename makes a different distinction than I had in mind, but a better one. Accordingly, I renamed this one and the existing occurrences of `rawArgs`, and one more, to `syncArgs`. Reads much better. Done.